controllers/articles.js

```diff
@@ -34,4 +34,70 @@ export default class ArticleController {
       return res.status(500).json({ success: false, error: [error.message] });
     }
   }
+
+  /**
+ * @description - Update an article
+ * @static
+ * @param {Object} req - the request object
+ * @param {Object} res - the response object
+ * @memberof ArticleController
+ * @returns {Object} class instance
+ */
+  static async updateArticle(req, res) {
+    const images = req.images || [];
+    const {
+      title, description, body
+    } = req.body;
+    const { id } = req.user;
+    try {
+      const result = await Article.update({
+        title,
+        description,
+        body,
+        images,
+        userId: id
+      }, {
+        where: {
+          id: req.params.id
+        }
+      });
+      return res.status(200).json({
+        success: true,
+        message: 'Article updated successfully',
+        article: result
+      });
+    } catch (error) {
+      return res.status(500).json({
+        success: false,
+        errors: [error.message]
+      });
+    }
+  }
+
+  /**
+ * @description - Update an article
```

Comment:
Please review this, should be `Delete an article`

Reply:
Alright. Thanks

Reply:
Done.